buildprocess/generate-init-routes.js

```diff
@@ -0,0 +1,53 @@
+var fs = require("fs");
+var path = require("path");
+var URI = require("urijs");
+// var CATALOG_ROUTE = require("terriajs/lib/ReactViewModels/TerriaRouting").CATALOG_ROUTE;
+var CATALOG_ROUTE = `/catalog/`;
+
+// const rootGroupName = "Root Group";
+const rootGroupName = "/";
+
+const getNameFromItem = item => item.name;
+const getIdFromItem = item => item.id;
+
+
+const getRoutes = (topCatalogItem, currentRoute = rootGroupName) =>
+  topCatalogItem.reduce((acc, catalogItem) => {
+    const nameForCurrentItem = getNameFromItem(catalogItem);
+    // Id is used for uniqueid, so disregard any pathing
+    const idFromItem = getIdFromItem(catalogItem);
+    const finalRoute = idFromItem || `${currentRoute}/${nameForCurrentItem}`;
+    const items = catalogItem.items && getRoutes(catalogItem.items, finalRoute);
```

Comment:
This should be `catalog.members` following the `v8` schema.

Reply:
Ah yes, good pickup!